lab-cpp_5-10_function.cpp

```diff
@@ -0,0 +1,89 @@
+#include <iostream> 		// Підключення стандартної бібліотеки воду/виводу 
+#include "lab-cpp_5-10_class.h"	// Підключення власної бібліотеки з класами для о'бєктів файлу і тек 
```

Comment:
reading name of heading, I have to imagine what's inside. "lab-cpp_5-10_class" say to me nothing